web/frontend/src/components/slam.vue

```diff
@@ -183,6 +183,9 @@ const executeMoveOnMap = async () => {
 const refresh2d = async (name: string) => {
   const map = await fetchSLAMMap(name);
   const returnedPose = await fetchSLAMPose(name);
+  returnedPose.setX(returnedPose.getX() / 1000);
+  returnedPose.setY(returnedPose.getY() / 1000);
+  returnedPose.setZ(returnedPose.getZ() / 1000);
```

Comment:
@kim-mishra , could you add a check for a feature flag in the GetPosition response and use it to decide whether to multiply by 1000?